src/index.ts

```diff
@@ -34,21 +37,63 @@ export class ChainOps {
     }
   }
 
-  async getGasPrice(blockNumber?: number) {
+  async getGasPrice (blockNumber?: number) {
     return oracle.getGasPrice(this.getEndpoint('ORACLE_URL'), blockNumber)
   }
 
-  getEndpoint(endpointName: string): string {
-    //@ts-ignore
+  getEndpoint (endpointName: string): string {
+    // @ts-ignore
     if (!this.config[endpointName] || this.config[endpointName].length === 0) {
       throw new Error(endpointName + ' endpoint not defined')
     }
 
-    //@ts-ignore
+    // @ts-ignore
     return this.config[endpointName]
   }
 
-  async subscribe(subConfig: any) {
+  async getOptimisticBalance (wallet: EthAddress, tokenContract: EthAddress) {
```

Comment:
❔ Would it be useful to include a docstring describing the purpose of this function, and why it's called "optimistic" balance?